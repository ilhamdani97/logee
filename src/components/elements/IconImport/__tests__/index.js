import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import IconImport from '../IconImport';

describe('src/components/elements/IconImport', () => {
  test('render', () => {
    const shallow = new ShallowRenderer();
    const tree = shallow.render(<IconImport   />);
    expect(tree).toMatchSnapshot();
  });
});
