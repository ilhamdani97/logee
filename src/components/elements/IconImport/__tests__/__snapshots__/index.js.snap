// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/components/elements/IconImport render 1`] = `
<svg
  fill="none"
  height="16"
  viewBox="0 0 16 16"
  width="16"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    clipRule="evenodd"
    d="M7.74671 12.8598C7.62133 12.9879 7.45235 13.064 7.27338 13.0731C6.42671 13.0731 5.73338 11.2331 5.60004 8.82648H8.00004L10.3134 6.49315C10.3907 6.41135 10.4781 6.33967 10.5734 6.27981C10.4851 5.40256 10.3018 4.53749 10.0267 3.69981C10.62 4.08419 11.1263 4.5883 11.5134 5.17981C11.7009 5.44904 11.8638 5.73464 12 6.03315C12.3115 6.10566 12.5958 6.26519 12.82 6.49315C13.1433 6.82226 13.3245 7.26515 13.3245 7.72648C13.3245 8.18781 13.1433 8.6307 12.82 8.95981L12.5534 9.22648H13.9334C14.0163 8.8227 14.0609 8.41199 14.0667 7.99981C14.0628 7.05359 13.8613 6.1186 13.4751 5.25479C13.0888 4.39097 12.5264 3.61736 11.8238 2.98355C11.1212 2.34973 10.2939 1.86968 9.39504 1.57417C8.49613 1.27865 7.5454 1.17419 6.60377 1.26746C5.66215 1.36074 4.75039 1.6497 3.92693 2.11583C3.10347 2.58196 2.38647 3.21498 1.82188 3.97433C1.25729 4.73367 0.857557 5.60258 0.6483 6.52539C0.439044 7.4482 0.424879 8.40455 0.606712 9.33315C0.869284 10.6496 1.516 11.8588 2.46521 12.808C3.41442 13.7572 4.62358 14.4039 5.94004 14.6665C7.06144 14.8807 8.21857 14.812 9.30671 14.4665L7.74671 12.8598ZM7.27338 2.85981C8.11338 2.85981 8.80671 4.69981 8.94671 7.10648H5.61338C5.73338 4.71981 6.42671 2.87981 7.27338 2.87981V2.85981ZM3.01338 10.7731C2.62611 10.1848 2.36716 9.52155 2.25338 8.82648H3.90004C3.93606 9.98925 4.1381 11.1409 4.50004 12.2465C3.90856 11.8627 3.40249 11.3611 3.01338 10.7731ZM3.90004 7.12648H2.25338C2.36662 6.43019 2.62807 5.76633 3.02005 5.17981C3.40469 4.5886 3.90883 4.08446 4.50004 3.69981C4.13747 4.80753 3.93542 5.96147 3.90004 7.12648Z"
    fill="#757575"
    fillRule="evenodd"
  />
  <path
    clipRule="evenodd"
    d="M14.8667 9.91262H10.9333L12.32 8.50595C12.5223 8.30344 12.636 8.02888 12.636 7.74262C12.636 7.45635 12.5223 7.1818 12.32 6.97928C12.2212 6.87768 12.1031 6.79692 11.9725 6.74177C11.842 6.68662 11.7017 6.6582 11.56 6.6582C11.4183 6.6582 11.278 6.68662 11.1475 6.74177C11.0169 6.79692 10.8988 6.87768 10.8 6.97928L7.58 10.226C7.58 10.226 7.58 10.226 7.58 10.2593C7.48949 10.3517 7.41713 10.4602 7.36667 10.5793C7.36352 10.5902 7.36352 10.6017 7.36667 10.6126C7.32161 10.7203 7.29893 10.8359 7.3 10.9526C7.29665 10.9881 7.29665 11.0238 7.3 11.0593C7.30093 11.1514 7.31211 11.243 7.33333 11.3326C7.34483 11.3688 7.35818 11.4044 7.37333 11.4393C7.42546 11.5575 7.49764 11.6657 7.58667 11.7593L10.8067 15.006C10.9055 15.1076 11.0236 15.1883 11.1541 15.2435C11.2847 15.2986 11.425 15.327 11.5667 15.327C11.7084 15.327 11.8486 15.2986 11.9792 15.2435C12.1097 15.1883 12.2279 15.1076 12.3267 15.006C12.529 14.8034 12.6426 14.5289 12.6426 14.2426C12.6426 13.9564 12.529 13.6818 12.3267 13.4793L10.9333 12.0726H14.8667C15.1531 12.0726 15.4278 11.9588 15.6303 11.7563C15.8329 11.5538 15.9467 11.2791 15.9467 10.9926C15.9467 10.7062 15.8329 10.4315 15.6303 10.2289C15.4278 10.0264 15.1531 9.91262 14.8667 9.91262Z"
    fill="#757575"
    fillRule="evenodd"
  />
</svg>
`;
